3/3.11/download_source

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -eu
```

Comment:
We should file a bug to track this for now. Checksumming each file is a bit too heavy weighted imho.